<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myMainWindow</class>
 <widget class="QMainWindow" name="myMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>myMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QToolBox" name="QTreeWidget">
       <property name="minimumSize">
        <size>
         <width>163</width>
         <height>357</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>163</width>
          <height>269</height>
         </rect>
        </property>
        <attribute name="label">
         <string>QListWidget操作</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QToolButton" name="toolButton">
             <property name="minimumSize">
              <size>
               <width>123</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>tBtnListIni</string>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextOnly</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QToolButton" name="toolButton_2">
             <property name="minimumSize">
              <size>
               <width>123</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>tBtnListClear</string>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextOnly</enum>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QToolButton" name="toolButton_3">
             <property name="minimumSize">
              <size>
               <width>123</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>tBtnListInsert</string>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextOnly</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QToolButton" name="toolButton_4">
             <property name="text">
              <string>tBtnListAppend</string>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextOnly</enum>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QToolButton" name="toolButton_5">
             <property name="minimumSize">
              <size>
               <width>123</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>tBtnListDelete</string>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextOnly</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>163</width>
          <height>269</height>
         </rect>
        </property>
        <attribute name="label">
         <string>QTreeWidget</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="page_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>163</width>
          <height>269</height>
         </rect>
        </property>
        <attribute name="label">
         <string>QTableWidget</string>
        </attribute>
       </widget>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="minimumSize">
        <size>
         <width>351</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>QListWidget</string>
        </attribute>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>12</x>
           <y>16</y>
           <width>473</width>
           <height>309</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string> 当前变化项</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="editCutItemText"/>
            </item>
            <item>
             <widget class="QCheckBox" name="chkBoxListEditable">
              <property name="font">
               <font>
                <family>微软雅黑</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string> 可编辑</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QToolButton" name="tBtnSelectItem">
              <property name="text">
               <string>tBtnSelectItem</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="tBtnSelALL">
              <property name="text">
               <string>tBtnSelALL</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="tBtnSelNone">
              <property name="text">
               <string>tBtnSelNone</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="tBtnSelInvs">
              <property name="text">
               <string>tBtnSelInvs</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget">
            <item>
             <property name="text">
              <string>New Item</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>New Item</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>QTreeWidget</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>QTableWidget</string>
        </attribute>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actListIni"/>
   <addaction name="actListClear"/>
   <addaction name="actListInsert"/>
   <addaction name="actListDelete"/>
  </widget>
  <action name="actListIni">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/image/image/toolbarlab.jfif</normaloff>:/image/image/toolbarlab.jfif</iconset>
   </property>
   <property name="text">
    <string>初始化列表</string>
   </property>
  </action>
  <action name="actListClear">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/image/image/cut.jfif</normaloff>:/image/image/cut.jfif</iconset>
   </property>
   <property name="text">
    <string>清除列表</string>
   </property>
  </action>
  <action name="actListInsert">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/image/image/new.jfif</normaloff>:/image/image/new.jfif</iconset>
   </property>
   <property name="text">
    <string>插入项</string>
   </property>
  </action>
  <action name="actListDelete">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/image/image/redX.jpg</normaloff>:/image/image/redX.jpg</iconset>
   </property>
   <property name="text">
    <string>删除当前项</string>
   </property>
  </action>
  <action name="actSelALL">
   <property name="text">
    <string>全选</string>
   </property>
  </action>
  <action name="actSellNone">
   <property name="text">
    <string>全不选</string>
   </property>
  </action>
  <action name="actSelInvs">
   <property name="text">
    <string>反选</string>
   </property>
  </action>
  <action name="actSelPopMenu">
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/image/image/open.jfif</normaloff>:/image/image/open.jfif</iconset>
   </property>
   <property name="text">
    <string>项选择</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>toolButton_2</sender>
   <signal>clicked()</signal>
   <receiver>actListClear</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_3</sender>
   <signal>clicked()</signal>
   <receiver>actListInsert</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_5</sender>
   <signal>clicked()</signal>
   <receiver>actListDelete</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
